src/main/java/com/yahoo/bullet/rest/service/WebSocketService.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 2018, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.rest.service;
+
+import com.yahoo.bullet.pubsub.Metadata;
+import com.yahoo.bullet.rest.model.WebSocketResponse;
+import com.yahoo.bullet.rest.query.WebSocketQueryHandler;
+import lombok.AccessLevel;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.simp.SimpMessageHeaderAccessor;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Service
+public class WebSocketService {
+    private static final String DESTINATION =  "/response/private";
```

Comment:
Probably constructed from the "response" defined in WebSocketConfig

Reply:
Yes. And also moved /private to application.yaml